src/accounts/AccountPage.tsx

```diff
@@ -103,7 +63,6 @@ const AccountAboutPage: FC = () => {
   }
 
   const showDeactivateAccountSection = isFlagEnabled('freeAccountCancellation')
-  const showLeaveAcctBtn = !isFlagEnabled('createDeleteOrgs')
```

Comment:
TODO: check if this feature needs to remain behind a feature flag

Reply:
if (`createDeleteOrgs`) Leave Account becomes Leave Org and gets moved to Orgs setting page 
if (!`createDeleteOrgs`) Leave Account remains in Accounts settings page